src/Api/Controllers/AccountsController.cs

```diff
@@ -194,6 +195,30 @@ public async Task PostPassword([FromBody]PasswordRequestModel model)
             await Task.Delay(2000);
             throw new BadRequestException(ModelState);
         }
+        
+        [HttpPost("password-no-compare")]
+        public async Task PostPasswordNoCompare([FromBody]PasswordNoCompareRequestModel model)
+        {
+            var user = await _userService.GetUserByPrincipalAsync(User);
+            if (user == null)
+            {
+                throw new UnauthorizedAccessException();
+            }
+
```

Comment:
We need to validate that the user does not have a password already set. If so, they would need to use the normal change password API that requires comparison of the existing password.

Reply:
Updated with user has password error flow.